templates/zerver/api/server-settings.md

```diff
@@ -0,0 +1,70 @@
+# Get global settings
+
+Fetch global settings for the Zulip server.
+
+`GET {{ api_url }}/v1/server_settings`
+
```

Comment:
Let's also give a sentence or two about what it's intended to be used for -- it's a no-authentication-required query you can send to a server to check (1) if it's a zulip server and (2) what you need to know to display a logic prompt for it (e.g. what authentication methods are available).

Reply:
(We should write such docs for anything where it isn't obvious, like sending a message)

Reply:
I get the auth methods example, but I'm not 100% sure about what you mean with the "logic prompt". I've just borrowed your words for now, while I get this straight.

Reply:
I meant "login prompt" :)